website/source/docs/commands/connect/envoy.html.md.erb

```diff
@@ -65,6 +65,30 @@ proxy configuration needed.
   service](/docs/connect/registration/service-registration.html) ID on the
   local agent. This must already be present on the local agent.
 
+* `-mesh-gateway` - Flag to indicate that Envoy should be configured as a Mesh
```

Comment:
I think the idea we talked about here was to break this into sections for Sidecar Only, Gateway Only, and Sidecar or Gateway settings. This will probably help to make things more clear and prevent new users from having to skim through the whole section to find the settings for their proxy type.